CHANGELOG/CHANGELOG-1.15.md

```diff
@@ -0,0 +1,77 @@
+# Changelog 1.15
+
+## 1.15.0 - 2024-01-26
+
+### Added
+
+- Add a new feature `NodeNetworkPolicy` to allow user to apply `ClusterNetworkPolicy` to Kubernetes Nodes. ([#5658](https://github.com/antrea-io/antrea/pull/5658) [#5716](https://github.com/antrea-io/antrea/pull/5716), [@hongliangl] [@Atish-iaf])
+- Add L7 network flow export support in Antrea to enable users to export L7 protocol information. ([#5218](https://github.com/antrea-io/antrea/pull/5218), [@tushartathgur])
+- Add migration tool to support migrating from other CNIs to Antrea. ([#5677](https://github.com/antrea-io/antrea/pull/5677), [@hjiajing])
+- Support Egress using IPs from a separate subnet through VLAN. ([#5799](https://github.com/antrea-io/antrea/pull/5799), [@tnqn])
+- Record an event when Egress IP assignment changes for better troubleshooting. ([#5765](https://github.com/antrea-io/antrea/pull/5765), [@jainpulkit22])
+- Add Support for `FlexibleIPAM` on Antrea Multicast so that multicast packets can be forwarded to the uplink interface when `FlexibleIPAM` is enabled. ([#4922](https://github.com/antrea-io/antrea/pull/4922), [@ceclinux])
```

Comment:
We don't have a feature gate called FlexibleIPAM, and the words starting with "so that" is perhaps too implementation specific.
Perhaps just say "Add Antrea flexible IPAM support for the Multicast feature"

Reply:
Updated